grails-app/assets/javascripts/payer/payerSave.js

```diff
@@ -0,0 +1,42 @@
+$(document).ready(function () {
+  $("form").on("submit", function (e) {
+    e.preventDefault();
+
+    var data = new FormData(document.querySelector("form"));
+    var payer = {};
+
+    data.forEach(function (value, key) {
+      payer[key] = value;
+    });
+
+    var url = $("form").prop("action");
+
+    $.post(url, payer, function (response) {
```

Comment:
```suggestion
    $.post(url, params, function(response) {
```
Procurem padronizar dessa forma os posts que fazerem, até em questão de espaçamento